woof-code/rootfs-skeleton/etc/rc.d/rc.sysinit

```diff
@@ -455,39 +455,44 @@ if [ $PUPMODE -eq 2 ] ; then #full hd installation.
 fi
 
 [ ! -d /proc/acpi ] && modprobe apm #load apm.ko if no acpi.
-zcat /proc/config.gz | grep -m 1 -q '^CONFIG_INPUT_LEDS=m' && modprobe input-leds #for >= k4.2x
+modprobe input-leds 2>/dev/null #for >= k4.2x
```

Comment:
Loading `input-leds` if `CONFIG_INPUT_LEDS=m` is the same as trying to load `input-leds` and letting it fail if `CONFIG_INPUT_LEDS=y` or `CONFIG_INPUT_LEDS=n`, but much slower, because modprobe uses the cache generated by depmod (which includes a list of builtin modules and .ko paths) and module lookup is very fast - faster than decompressing /proc/config.gz